src/miner.cpp

```diff
@@ -143,7 +143,7 @@ CBlockTemplate* CreateNewBlock(const CChainParams& chainparams, const CScript& s
       CBlockHeader h;
       nBlockSize += h.GetSerializeSize(SER_NETWORK, PROTOCOL_VERSION);
     }
-    assert(nBlockSize == 80);  // BU always 80 bytes
+    assert(nBlockSize == 80);  // BU block header is always 80 bytes
```

Comment:
Putting comment on their own line make diff/history/blame clearer has changing the comment doesn't appear as changing the code.